File: src/PlatypusMeat.cpp
#include "../include/PlatypusMeat.hpp"

//Constructor menerima harga, dan mempassing producer untuk superclass.
PlatypusMeat::PlatypusMeat(int _price, string _producer){
	initializeFarmProduct(_producer);
	initializeProduct(_price, FARMPRODUCT, PLATYPUS_MEAT, '?');
}

File: Java/src/main/java/engisubs/gameobject/product/farm_product/meat/PlatypusMeat.java
package main.java.engisubs.gameobject.product.farm_product.meat;

public class PlatypusMeat extends Meat{
    public PlatypusMeat(int _price, String _producer){
        super.initializeFarmProduct(_producer);
        super.initializeProduct(_price, ProductType.FARMPRODUCT, GameObjectType.PLATYPUS_MEAT, "Daging Segar Platypus");
    }
}